tests/integration/targets/hcloud_primary_ip_info/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+# Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: setup ensure primary ip is absent
+  hcloud_primary_ip:
+    name: "{{ hcloud_primary_ip_name }}"
+    state: absent
+
+- name: setup primary ip
+  hcloud_primary_ip:
+    name: "{{ hcloud_primary_ip_name }}"
+    datacenter: "fsn1"
```

Comment:
fsn1 is the name of the `location`, the datacenter is called `fsn1-dc14`. This causes the CI to fail: https://dev.azure.com/ansible/hetzner.hcloud/_build/results?buildId=80952&view=logs&j=a994a8d5-c6f5-591c-c725-087e775e10a9&t=c873beaa-7658-5701-2101-3923fa72aaae&l=728